services/Donkey/src/donkey/core.clj

```diff
@@ -219,13 +216,21 @@
    :art-id "donkey"
    :service "donkey"})
 
+(def exception-handlers
+  {:handlers {::ex/request-validation (as-de-exception-handler ex/request-validation-handler ec/ERR_ILLEGAL_ARGUMENT)
+              :invalid-configuration (as-de-exception-handler invalid-cfg-handler ec/ERR_CONFIG_INVALID)
+              ::ex/response-validation (as-de-exception-handler ex/response-validation-handler ec/ERR_SCHEMA_VALIDATION)
+              ::ex/default unchecked-handler}})
+
 (defn site-handler
   [routes-fn]
   (-> (delayed-handler routes-fn)
-    wrap-keyword-params
-    wrap-lcase-params
-    wrap-query-params
-    (tc/wrap-thread-context svc-info)))
+      (wrap-exceptions exception-handlers)
+      wrap-logging
+      wrap-user-info
+      wrap-keyword-params
+      wrap-lcase-params
+      wrap-query-params))
```

Comment:
@psarando  
`(wrap-query-params (wrap-lcase-params (wrap-keyword-params (wrap-user-info (wrap-logging (wrap exceptions (delayed-handler routes-fn)))))))`
This is how I understand the threading macro would expand the above. So, the user info should be executed before both the logging and the exception handling.

https://clojuredocs.org/clojure.core/-%3E